src/command/cmd_defs.c

```diff
@@ -1643,12 +1643,12 @@ static const struct cmd_t command_defs[] = {
               "Add, remove or list command aliases.")
       CMD_ARGS(
               { "list", "List all aliases." },
-              { "add <name> <value>", "Add a new command alias." },
+              { "add <name> <value>", "Add a new command alias. The alias name should not contain any space characters." },
```

Comment:
I think it needs to be "must not" since it's not allowed/possible to have them in the alias *name* but it's possible to use them later on.
To make this more clear it would be good to add one more example.
Maybe readd the away example. The `/away` command just moved to `/status set away`